src/redux/actions/user.actions.ts

```diff
@@ -1,11 +1,3 @@
-import { UserState, Institution, SET_USER, UPDATE_INSTITUTION } from 'redux/types/user.types'
+import { UserState, SET_USER } from 'redux/types/user.types'
```

Comment:
Why is `SET_USER` not camelcase? Is there another specific convention for reducers?

Reply:
It's a redux convention to name redux types all uppercase yes.